Screen-Sharing/Screen-Sharing/ScreenCapturer.swift

```diff
@@ -94,21 +102,22 @@ class ScreenCapturer: NSObject, OTVideoCapture {
     func start() -> Int32 {
         capturing = true
         captureQueue.sync {
-            timer.resume()
+            resume()
         }
         return 0
     }
     
     func stop() -> Int32 {
         capturing = false
         captureQueue.sync {
-            timer.cancel()
+            suspend()
         }
         return 0
     }
     
     func releaseCapture() {
         timer.cancel()
+        resume()
```

Comment:
Shouln't it be `suspend()`?

Reply:
Suspend is used when the capturer is active, here the capturer is released, so we want to release the timer.
To deallocate a timer, it needs to be cancelled. If this is not done, GCD will trigger the timer and call the event handler on a deallocated object and crash. We also don’t want to cancel a timer that has been suspended because this will also trigger a crash.
This isn’t really well documented but I did find a post in Official Forums:
https://developer.apple.com/forums/thread/15902

Reply:
Ah, perfect! Thank you for the detailed explanation